jmxtrans-output/jmxtrans-output-elastic/src/main/java/com/googlecode/jmxtrans/model/output/elastic/ElasticWriter.java

```diff
@@ -128,7 +128,7 @@ protected void internalWrite(Server server, Query query, ImmutableList<Result> r
 					map.put("key", values.getKey());
 					map.put("keyAlias", result.getKeyAlias());
 					map.put("value", Double.parseDouble(value.toString()));
-					map.put("timestamp", result.getEpoch());
+					map.put("timestamp", java.util.Date(result.getEpoch()));
```

Comment:
@lmx1989219: No problem! You are trying to fix something that is broken, I applaud your efforts! It is non trivial to fix it without breaking it for users who are relying on the current behavior, but that's the fun part!